mava/logger.py

```diff
@@ -90,6 +90,18 @@ def log(
             f"Max Episode Length {float(np.max(episodes_length)):.3f}"
         )
 
+        # Added for LBF:
+        if trainer_metric and "LevelBasedForaging" in config["env_name"]:
+            num_eaten = jnp.ravel(episodes_info["num_eaten"])
+            percent_eaten = jnp.ravel(episodes_info["percent_eaten"])
+            logger.log_stat("mean_num_eaten", float(np.mean(num_eaten)), t_env)
+            logger.log_stat("mean_percent_eaten", float(np.mean(percent_eaten)), t_env)
+            log_string = (
+                log_string
+                + f" | Num Eaten {float(np.mean(num_eaten)):.3f} | "
+                + f"Percent Eaten {float(np.mean(percent_eaten)):.3f}"
+            )
```

Comment:
I really like this! 

However it should be done in the env and then we can grab it and log it here so that we don't have to make if statements for each different env. See how we add these kind of statistics in connector [here](https://github.com/instadeepai/jumanji/blob/main/jumanji/environments/routing/connector/env.py#L272-L283) and [here](https://github.com/instadeepai/jumanji/blob/main/jumanji/environments/routing/connector/env.py#L181-L194)

So then we could change add extras to our episode metrics and always log extras.

Reply:
Done :heavy_check_mark: Please take a look at it :)